<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>942</width>
    <height>742</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Словарь для тестов</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>901</width>
      <height>671</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="AddGroupAndWord">
     <attribute name="title">
      <string>Добавление слов и групп</string>
     </attribute>
     <widget class="QLineEdit" name="lineEditWord">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>60</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>29</x>
        <y>63</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Слово:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditTranslate">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>60</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>63</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Перевод:</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidgetWordTranslate">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>371</width>
        <height>481</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxGroups">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>10</y>
        <width>221</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonAddWord">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить слово</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonDeleteWord">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>100</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Удалить слово</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>13</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Группа:</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidgetGroup">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>140</y>
        <width>371</width>
        <height>481</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditGroup">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>60</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>62</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Введите имя группы:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonAddGroup">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>100</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить группу</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonDeleteGroup">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>100</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Удалить группу</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonSaveChanges">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>131</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Сохранить изменения</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabTest">
     <attribute name="title">
      <string>Тестирование</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>151</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Тип теста</string>
      </property>
      <widget class="QRadioButton" name="radioButtonEnterTranslate">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>121</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Ввод перевода</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButtonEnterWord">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>121</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Ввод слова</string>
       </property>
      </widget>
     </widget>
     <widget class="QComboBox" name="comboBoxGroupToTest">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>17</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Выбор группы для теста:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonStartTest">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>182</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Начать тест</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>230</y>
        <width>331</width>
        <height>391</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="labelWordOrTranslate">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>201</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Слово:</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelWordOrTranslateValue">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>96</y>
         <width>301</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Слово</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelEnterWordOrTranslate">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>201</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Введите перевод:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditWordOrTranslate">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>311</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonAnswer">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>230</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Ответить</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelCurrentNum">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>00</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelCurrentNum_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>20</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>из</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelCount">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>20</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>00</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonEnd">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>290</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Закончить</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelStateTest">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>340</y>
         <width>291</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Состояние теста</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelStateAnswer">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>190</y>
         <width>301</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonNextQuestion">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>230</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Cледующий</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="labelWordOrTranslate_2">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>190</y>
        <width>201</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Результаты:</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowserResuslt">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>230</y>
        <width>461</width>
        <height>391</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowserStatistics">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>20</y>
        <width>461</width>
        <height>161</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>942</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
